.github/workflows/lint.yaml

```diff
@@ -0,0 +1,22 @@
+---
+name: Linters
+
+on:  # yamllint disable-line rule:truthy
+  push:
+    branches: ['**']
+  pull_request:
+    branches: ['main']
+
```

Comment:
extra white space

Reply:
Fair enough, let's wack the white-space!